onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -49,14 +53,14 @@ class TreeEnsembleCommon {
                      const std::vector<int64_t>& target_class_treeids,
                      const std::vector<OTYPE>& target_class_weights);
 
-  void compute(const Tensor* X, Tensor* Z, Tensor* label) const;
+  void compute(concurrency::ThreadPool* ttp, const Tensor* X, Tensor* Z, Tensor* label) const;
 
  protected:
   TreeNodeElement<OTYPE>* ProcessTreeNodeLeave(
       TreeNodeElement<OTYPE>* root, const ITYPE* x_data) const;
 
   template <typename AGG>
-  void compute_agg(const Tensor* X, Tensor* Z, Tensor* label, const AGG& agg) const;
+  void compute_agg(concurrency::ThreadPool* ttp, const Tensor* X, Tensor* Z, Tensor* label, const AGG& agg) const;
```

Comment:
Something to fix later - function names don't conform to coding style.

Reply:
ComputeAgg I assume?

Reply:
yup

Reply:
Done